master/master.cfg

```diff
@@ -422,6 +425,50 @@ def get_msvc_config_steps(factory, builder_type):
                                extract_fn=save_interesting_env_vars))
 
 
+EMSDK_ENV_VARS = [
+    "EM_CACHE",
+    "EM_CONFIG",
+    "EMSDK",
+    "EMSDK_NODE",
+    "EMSDK_PYTHON",
+    "PATH",
+]
+
+def get_emsdk_config_steps(factory, builder_type):
+    if not builder_type.handles_wasm():
+        print("WARNING: should not call get_emsdk_config_steps() for builds that cannot handle WASM")
+        return
+
+    def save_interesting_env_vars(rc, stdout, stderr):
+        d = {}
+        for line in stdout.split('\n'):
+            match = re.match("^([a-zA-Z0-9_-]+) = (.*)$", line.strip())
+            if match:
+                key = match.group(1).upper()
+                value = match.group(2)
+                if key in [EMSDK_ENV_VARS]:
```

Comment:
This should be `if key in EMSDK_ENV_VARS`. No surrounding `[ .. ]`.

Reply:
Whoops, I knew that, typo